File: TransCoder_tokenized_test_set_functions/python/PRINT_NUMBER_ASCENDING_ORDER_CONTAINS_1_2_3_DIGITS.py
def printNumbers ( numbers ) :
    numbers = map ( str, numbers )
    result = [ ]
    for num in numbers :
        if ( '1' in num and '2' in num and '3' in num ) :
            result . append ( num )
    if not result :
        result = [ '-1' ]
    return sorted ( result );

File: results/java_codex/PRINT_NUMBER_ASCENDING_ORDER_CONTAINS_1_2_3_DIGITS.java
string printNumbers(vector<int> numbers) {
  vector<int> array;
  for (int number : numbers) {
    if (findContainsOneTwoThree(number)) array.push_back(number);
  }
  sort(array.begin(), array.end());
  string strbuf;
  for (int value : array) {
    if (strbuf.length() > 0) strbuf += ", ";
    strbuf += to_string(value);
  }
  return (strbuf.length() > 0) ? strbuf : "-1";
}